src/System.Private.CoreLib/shared/System/Runtime/Loader/AssemblyLoadContext.cs

```diff
@@ -509,9 +509,13 @@ public ContextualReflectionScope EnterContextualReflection()
         /// </remarks>
         public static ContextualReflectionScope EnterContextualReflection(Assembly activating)
         {
-            return activating != null ?
-                GetLoadContext(activating).EnterContextualReflection() :
-                new ContextualReflectionScope(null);
+            AssemblyLoadContext? assemblyLoadContext = (activating != null) ?
```

Comment:
Would it be better to throw here?

- `ArgumentNullException` for null activating assembly
- `ArgumentException(SR.Arg_MustBeType, nameof(activating));` for non-runtime assembly.

Reply:
>  `ArgumentException(SR.Arg_MustBeType, nameof(activating));` for non-runtime assembly.

Sure, but should be `SR.Arg_MustBeRuntimeAssembly` which needs to be added...
 `ArgumentException(SR.Arg_MustBeRuntimeAssembly, nameof(activating));` for non-runtime assembly.

Reply:
Yes, that's what I meant...